### PR TITLE
chore: graceful handling of notes with sub-lists

### PR DESCRIPTION
**Pull request by ckerr**
#### Description of Change

Handle multine release notes that contain their own bullet points.

Also, if a release note begins with a bullet point, remove it because it will confuse the markdown parser to have two bullet points.

Before and After:

---

* * Fixes the following issues for frameless when maximized on Windows: * fix unreachable task bar when auto hidden with position top * fix 1px extending to secondary monitor * fix 1px overflowing into taskbar at certain resolutions * fix white line on top of window under 4k resolutions. [#25216](https://github.com/electron/electron/pull/25216) <span style="font-size:small;">(Also in [8](https://github.com/electron/electron/pull/25219), [9](https://github.com/electron/electron/pull/25218), [11](https://github.com/electron/electron/pull/25217))</span>

---

* Fixed the following issues for frameless when maximized on Windows:
  * fix unreachable task bar when auto hidden with position top
  * fix 1px extending to secondary monitor
  * fix 1px overflowing into taskbar at certain resolutions
  * fix white line on top of window under 4k resolutions. [#25216](https://github.com/electron/electron/pull/25216) <span style="font-size:small;">(Also in [8](https://github.com/electron/electron/pull/25219), [9](https://github.com/electron/electron/pull/25218), [11](https://github.com/electron/electron/pull/25217))</span>
---

CC @codebytere @MarshallOfSound 

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).
- [x] This is **NOT A BREAKING CHANGE**. Breaking changes may not be merged to master until 11-x-y is branched.

#### Release Notes

Notes: none

### CODE REVIEW

**Comment by ckerr** [comment]
> @ckerr do you have a before and after example of formatting with bullet points?

@codebytere added in description

> I'd also prefer we add some tests if possible!

Added 3badf00ed8be3e11bc5715242a5916bb430fb440

> (lint is dying too)

Fixed bd296fed2bd5e4a7e9247fed14e9ff9cd9832707

**Review by codebytere — APPROVED** [review]
🎉

File: script/release/notes/notes.js [hunk 1]
```diff
@@ -6,10 +6,19 @@ const fs = require('fs');
...
 const { GitProcess } = require('dugite');
-const { Octokit } = require('@octokit/rest');
-const octokit = new Octokit({
-  auth: process.env.ELECTRON_GITHUB_TOKEN
-});
+
+let octokit;
+if (!octokit) {
+  const octokitParams = { auth: process.env.ELECTRON_GITHUB_TOKEN };
+  try {
```
Comment (MarshallOfSound) [inline script/release/notes/notes.js:13]: 😕  Why do we need this? This code will only run on the current branch, never across-branches

Comment (codebytere) [reply script/release/notes/notes.js:13]: Agreed ☝🏻  let's remove this, it should be whatever's necessary on that branch

Comment (ckerr) [reply script/release/notes/notes.js]: 868f90e86bd2f79cdb7be0199d48246776cee38f

**Review by MarshallOfSound — APPROVED** [review]

**Pull request merged**